main.go

```diff
@@ -12,11 +12,21 @@ import (
 
 	"github.com/google/uuid"
 	"golang.org/x/oauth2"
+
+	"io/ioutil"
+	"archive/zip"
+	"bytes"
 )
 
 const (
 	BOOK_COVER_PATH      = "public/books/image"
 	HIGHLIGHT_IMAGE_PATH = "public/highlights/image"
+	BOOK_EXPORT_PATH = "export/data/books.json"
+	HIGHLIGHT_EXPORT_PATH = "export/data/highlights.json"
+	SHELVES_EXPORT_PATH = "export/data/shelves.json"
+	USER_EXPORT_PATH = "export/data/user.json"
+	BOOK_IMAGES_EXPORT_PATH = "export/images/books"
+	HIGHLIGHT_IMAGE_EXPORT_PATH = "export/images/highlights"
 )
```

Comment:
there is a bit of a problem making the paths constants for all users. if two users are exporting their data at the same time that would overwrite each other files or worse one of the files would be open for write and another export fails. 
I would like to avoid writing to the disk completely